composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -1,73 +1,87 @@
+// External dependencies.
 import React from "react";
 import styled from "styled-components";
 import PropTypes from "prop-types";
+import uniqueId from "lodash/uniqueId";
+import { __ } from "@wordpress/i18n";
 
+// Internal dependencies.
 import colors from "../../../../style-guide/colors.json";
 
-let KeywordField = styled.input`
-	margin-right: 0.5em;
-	border-color: ${ props => props.borderColor };
+const errorColor = colors.$color_red;
+
+const KeywordInputContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	margin: 1em 0;
 `;
 
-const ErrorText = styled.div`
+const KeywordFieldLabel = styled.label`
 	font-size: 1em;
-	color: ${ colors.$color_red };
-	margin: 1em 0;
-	min-height: 1.8em;
+	font-weight: bold;
+	margin-bottom: 0.5em;
+`;
+
+const KeywordField = styled.input`
+	border: 3px solid ${ colors.$color_input_border };
+	padding: 0.75em;
+	font-size: 1em;
+
+	&.hasError {
+		border-color: ${ errorColor };
+		outline-color: ${ errorColor };
+	}
 `;
 
-const ErrorMessage = "Are you trying to use multiple keywords? You should add them separately below.";
+const ErrorText = styled.p`
+	color: ${ errorColor };
+	margin: 0.5em 0 0 0;
+	min-height: 1.8em;
+`;
 
 class KeywordInput extends React.Component {
 	/**
-	 * Constructs a KeywordInput component
+	 * Constructs a KeywordInput component.
 	 *
-	 * @param {Object}      props          The props for this input field component.
-	 * @param {String}    	props.id       The id of the KeywordInput.
-	 * @param {IconsButton} props.label    The label of the KeywordInput.
-	 * @param {boolean}     props.keyword  The initial keyword passed to the state.
+	 * @param {Object}   props           The props for this input field component.
```

Comment:
I'd vouch for being consistent with the usage of names... So instead of speaking of 'this input field component' I'd say 'The props for the KeywordInput'.